.github/workflows/release-bpmn_visualization.yml

```diff
@@ -37,7 +37,8 @@ jobs:
       - name: Checkout master
         run: git checkout master && git pull --tags
       - name: Bump Version
-        run: npm version ${{ github.event.inputs.type }} --message "chore(release) %s"
+        run: |
```

Comment:
**thought:** use `|` to avoid parsing error when using the colon character in the run command